### DISCUSSION TITLE
Can't make backrelation work in Pocketbase

### DISCUSSION THREAD

**Discussion by Mark-Vu** [pocketbase/pocketbase#4526] [Q&A] [answered]
I want to get all the ratings for a resume but i cant seem to make backrelation works, collection "ratings" have a field "resume" that is a relation to resumes: 
![image](https://github.com/pocketbase/pocketbase/assets/60988497/630f59c5-445b-4f8d-bca6-d7a4e566bd83)
![image](https://github.com/pocketbase/pocketbase/assets/60988497/972545e0-d0ca-4d4a-ba3f-8b09c2c65ccd)
![image](https://github.com/pocketbase/pocketbase/assets/60988497/3e5e9e30-ab06-4145-a564-e57f31023c9e)
![image](https://github.com/pocketbase/pocketbase/assets/60988497/6f77f5d0-a3ec-44a3-bb42-ef73e9621509)

OUTPUT (which dont have any ratings):
![image](https://github.com/pocketbase/pocketbase/assets/60988497/d0eb458c-a5d6-40d1-922f-891d1b0c53c6)

**Comment by ganigeorgiev** [comment]
It is difficult to say from just  the screenshots what could be the issue. I've tried to reproduce your collections structure locally and it works correctly to me. Make sure that you have an existing "ratings" record with relation to your queried "resumes".

If you are not able to resolve it, please post your collections configuration as json (_Admin UI > Settings > Export collections_).

**Reply by Mark-Vu** [reply, marked as answer]
I upgraded pocketbase to the newest version and it worked! Thank you so much for your answer

**Reply by ganigeorgiev** [reply]
Also I forgot to mention that the back-relation `_via_` `expand` syntax was introduced with PocketBase v0.22.0 so make sure that you use the latest version.
